engine/class_modules/sc_shaman.cpp

```diff
@@ -7461,11 +7466,19 @@ struct earthquake_overload_damage_t : public earthquake_damage_base_t
   earthquake_overload_damage_t( shaman_t* player, earthquake_base_t* parent ) :
     earthquake_damage_base_t( player, "earthquake_overload_damage", player->find_spell( 298765 ), parent )
   {
-    // Earthquake modifier is hardcoded rather than using effects, so we set the modifier here
-    spell_power_mod.direct = 0.3195 * player->talent.mountains_will_fall->effectN( 1 ).percent();
-    // TODO: implement spellpower coefficient extracttion from spell variable
-    // auto spell_desc = player->dbc->spell_desc_vars( this->data().id() );
-    // spell_desc.desc_vars()
+
+      if ( auto vars = player->dbc->spell_desc_vars( this->data().id() ).desc_vars() )
+      {
+          std::cmatch m;
+          std::regex get_var( R"(\$damage=\$\{\$SPN\*([\d\.]+)\*.*\})" );
+
+          if ( std::regex_search( vars, m, get_var ) )
+          {
+              auto coeff = util::to_double( m.str( 1 ) ) * player->talent.mountains_will_fall->effectN( 1 ).percent();
+              player->sim->print_debug( "parsed Earthquake Overload spell power coefficient to be '{}'", coeff );
```

Comment:
Player-relevant debug output generally contains the actor name in it, so you should add it to the debug message.